src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -30,14 +30,20 @@
           'ec-checkbox__check-icon-wrapper--focused': inputIsFocused,
           'ec-checkbox__check-icon-wrapper--checked': inputModel,
           'ec-checkbox__check-icon-wrapper--checked-and-focused': inputModel && inputIsFocused,
+          'ec-checkbox__check-icon-wrapper--indeterminate': indeterminate,
+          'ec-checkbox__check-icon-wrapper--indeterminate-and-focused': indeterminate && inputIsFocused,
```

Comment:
We also need a indeterminate-and-disabled style which should grey out the indeterminate state when disabled

Reply:
Good thinking, I will include those as well.